### DISCUSSION TITLE
Problem with duplicating the contents of the 'td' element

### DISCUSSION THREAD

**Discussion by wojciechkibitlewski** [rappasoft/laravel-livewire-tables#1323] [Q&A]
Hi,
When I go between paging the results, there is a problem with duplicating the contents of the 'td' element, created as follows:
Column::make('Actions')->label(fn($row, Column $column) => view('clients.table-actions')->withRow($row))->unclickable()

Have any of you encountered this? How to solve this problem?

![Zrzut ekranu 2023-08-31 001019](https://github.com/rappasoft/laravel-livewire-tables/assets/105292367/50e138c6-1faf-4abb-94f3-c634809c02b7)

![Zrzut ekranu 2023-08-31 001200](https://github.com/rappasoft/laravel-livewire-tables/assets/105292367/86442d42-5b40-49d6-bce9-87470b624c58)
![Zrzut ekranu 2023-08-31 001313](https://github.com/rappasoft/laravel-livewire-tables/assets/105292367/9e19457d-3232-4032-b4a9-794ce2c98a86)

**Comment by lrljoe** [comment]
What's the content of your view/blade file.

Can you confirm theme, laravel version, livewire version please
